dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.restfulWS-3.0.feature

```diff
@@ -8,7 +8,8 @@ IBM-Process-Types: client, \
  server
 Subsystem-Name: Jakarta RESTful Web Services 3.0
 -features=\
-  io.openliberty.jakarta.activation-2.0
+  io.openliberty.jakarta.activation-2.0, \
```

Comment:
Should we make a jaxrs-2.0 and 2.1 API feature with eeCompatible in it and have features depend on it instead of pulling in the bundle directly?

Reply:
> Should we make a jaxrs-2.0 and 2.1 API feature with eeCompatible in it and have features depend on it instead of pulling in the bundle directly?

Probably, will add to the separate issue for private features.